cedar-policy/src/api.rs

```diff
@@ -260,6 +261,56 @@ impl Entity {
             ancestors.into_iter().map(EntityUid::new).collect(),
         )
     }
+
+    /// Parse an entity from an in-memory JSON value
```

Comment:
Needs docs for how `schema` influences parsing. Maybe just a doc-link to `Entities::from_json_value`.